doc/parse_readmes.py

```diff
@@ -18,14 +18,14 @@
            os.environ['GITHUB_PASSWORD'])
 
 org = g.get_organization('binder-examples')
-repos = org.get_repos()
+repos = list(org.get_repos())
 
 lines = ['# Sample Binder Repositories\n',
          '\n'
          'Below we list several sample Binder repositories that\n'
          'demonstrate how to compose build files in order to create\n'
          'Binders with varying environments.\n\n']
-for repo in repos:
+for ii, repo in enumerate(repos):
```

Comment:
Let's define what `ii` is or likely even better pick a more meaningful name or abbreviation